pulp_smash/config.py

```diff
@@ -95,7 +95,7 @@ def _get_pulp_3_content_role():
     "anyOf": [
         {
             "additionalProperties": False,
-            "required": ["pulp", "hosts"],
+            "required": ["pulp", "general", "hosts"],
```

Comment:
I think this should not be required, or it will break compatibility with existing settings.

The general may be optional, if not preovided then it will use the default timeout config.